gensim/models/keyedvectors.py

```diff
@@ -1814,15 +1887,15 @@ def _load_word2vec_format(cls, fname, fvocab=None, binary=False, encoding='utf8'
 
 
 def load_word2vec_format(*args, **kwargs):
-    """Alias for `KeyedVectors.load_word2vec_format(...)`"""
+    """Alias for :meth:`~gensim.models.keyedvectors.KeyedVectors.load_word2vec_format`."""
     return KeyedVectors.load_word2vec_format(*args, **kwargs)
 
 
 def pseudorandom_weak_vector(size, seed_string=None, hashfxn=hash):
-    """Get a 'random' vector (but deterministically derived from seed_string if supplied).
+    """Get a random vector, derived deterministically from `seed_string` if supplied.
+
+    Useful for initializing KeyedVectors that will be the starting projection/input layers of _2Vec models.
```

Comment:
FYI/ this per-slot pseudorandom initialization is no longer used in #2944's changes, and thus this method will likely disappear entirely.

Reply:
OK. Let's merge this ASAP, I fear divergence. CC @mpenkov.